src/main/java/seedu/address/logic/commands/ShiftAddCommand.java

```diff
@@ -19,7 +19,7 @@
             + "Parameters: "
             + PREFIX_SHIFT_DAY + "DAY "
             + PREFIX_SHIFT_TIME + "TIME "
-            + "[" + PREFIX_ROLE_REQUIREMENT + "ROLE <SPACE> NUMBER_NEEDED]...\n"
+            + "[" + PREFIX_ROLE_REQUIREMENT + "ROLE <space> NUMBER_NEEDED]...\n"
```

Comment:
Perhaps remove `<space>` here as well. Same issue in `ShiftEditCommand.java` and `RoleRequirement.java`

Reply:
I have removed the <space> from the error messages, good catch